src/server.c

```diff
@@ -195,6 +191,8 @@ void handle_http_request(int fd, struct cache *cache)
  */
 int main(void)
 {
+    srand(time(0));
```

Comment:
jajajaaa and you set the seed, so it's not always the same sequence of random. nice!
great idea setting that here, so that this only happens once (when you run the server), instead of every time you call the d20